resources/ts/Components/Integrations/Detail/ContactInfo.tsx

```diff
@@ -166,7 +166,6 @@ export const ContactInfo = ({ id, contacts, isMobile }: Props) => {
           <Alert
             variant="error"
             title={t("details.contact_info.alert.title")}
-            closable
             onClose={() => setIsContactInfoAlertVisible(false)}
```

Comment:
If it's not closable anymore, the IsContactInfoAlertVisible state can also be removed, no?